parambokeh/__init__.py

```diff
@@ -209,7 +209,8 @@ def change_event(self):
         else:
             self._changed[p_name] = new_values
 
-        if self.p.mode == 'notebook' and self.p.push:
+        # document.hold() must have been done already? because this seems to work
```

Comment:
I just checked and the bokeh ``CommsHandle`` automatically put a hold on the document so this should be fine.